Userland/Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -428,6 +428,10 @@ void TerminalWidget::paint_event(GUI::PaintEvent& event)
             if (code_point == ' ')
                 continue;
 
+            if (has_flag(attribute.flags, VT::Attribute::Flags::Concealed)) {
+                continue;
+            }
```

Comment:
Wait, does this actually affect selection and copy-pasting? This is paint-only code here, right? Are you sure the patch as-is doesn't already work?

Reply:
Hey @nico , you are correct. 
![image](https://github.com/user-attachments/assets/423b9be9-6300-4e49-92a5-26b88889fe75)

My code as-is copies to clipboard, exhibiting the same behaviour of other terminals with concealed text.